tidb-cloud/size-your-cluster.md

```diff
@@ -182,6 +182,33 @@ The supported node storage of different TiKV vCPUs is as follows:
 >
 > You cannot decrease the TiKV node storage after the cluster creation.
 
+### TiKV node storage type
+
+The following TiKV storage types are provided for TiDB Cloud Dedicated clusters:
+
+#### Basic Type
+
+A general-purpose storage type which provides lower performance than the Standard Type.
+
+Usage： 
+- Clusters created before 2025/4/1 used Basic Type. 
+- New Clusters created with version < 7.5.5, 8.1.2, or 8.5.0 will use Basic Type. 
+
+#### Standard Type
+
+Well-suited for the majority of workloads due to its balanced performance and cost ratio.
+Standard storage type can improve the overall performance comparing to Basic type. It reserves sufficient disk resources for raft logs, reducing the performance impact between raft I/O and data disk I/O, thereby improving both the read and write performance of TiKV.
```

Comment:
```suggestion
The Standard Type is well-suited for the majority of workloads due to its balanced performance and cost ratio. It can improve the overall performance comparing to the Basic Type. It reserves sufficient disk resources for raft logs, reducing the performance impact between raft I/O and data disk I/O, thereby improving both the read and write performance of TiKV.
```